TaskClass.dart

```diff
@@ -0,0 +1,28 @@
+// Your job is to create a simple task list app using Dart. The app should allow users to add, edit, and delete tasks. The app should also allow users to mark tasks as completed. The app should be able to handle errors and display error messages to the user.
+
+
+class TaskInfo{
+  String Num = '';
+  String Name = '';
+  String Description = '';
+  bool CompletedOrNot = false;
+
+  TaskInfo(this.Num,this.Name,this.Description,this.CompletedOrNot);
```

Comment:
should use a more specific constructor
```suggestion
  TaskInfo({
    this.Num = "",
    this.Name = "",
    this.Description = "",
    this.CompletedOrNot = ""
    });
```